pkg/elevate/elevate.go

```diff
@@ -55,6 +55,19 @@ func RunElevate(argv []string) error {
 		return err
 	}
 
+	// As WriteKubeconfigToFile(utils.CreateTempKubeConfig) is overriding KUBECONFIG,
+	// we need to store its definition in order to redefine it to its original (value or unset) when we do not need it anymore
+	oldKubeconfigPath, oldKubeconfigDefined := os.LookupEnv("KUBECONFIG")
+	defer func() {
+		if oldKubeconfigDefined {
+			logger.Debugln("Will set KUBECONFIG variable to original", oldKubeconfigPath)
+			os.Setenv("KUBECONFIG", oldKubeconfigPath)
```

Comment:
any chance to add some unit tests to cover this change?
better to cover all login commands ( `login {cluster-id}` , `login --multi `  , `session {cluster-id}` )

Reply:
Sorry, but this is not a login command, this will be used only for elevate command